ui/src/app/icons/playground-mode-icon.tsx

```diff
@@ -1,30 +1,85 @@
 import * as React from 'react';
 
+import colors, {colorWithWhiteness} from 'app/styles/colors';
 
-export const PlaygroundModeIcon = () => <svg version='1.1'
-                                             id='Layer_1'
-                                             viewBox='0 0 14.1 12.3'>
-  <title>Group</title>
-  <desc>Created with Sketch.</desc>
-  <g id='ANAYLSIS'>
-    <g id='Notebook-PREVIEW' transform='translate(-363.000000, -85.000000)'>
-      <g id='Group-5' transform='translate(363.000000, 85.000000)'>
-        <g id='Group-3' transform='translate(0.889808, 0.297292)'>
-          <g id='Group' transform='translate(0.047675, 0.625027)'>
-            <path fill='#216FB4' d={`M11.8-0.9c0.7,0,1.3,0.6,1.3,1.3V10c0,0.7-0.6,1.3-1.3,1.3H0.4c
-            -0.7,0-1.3-0.6-1.3-1.3V0.4c0-0.7,0.6-1.3,1.3-1.3C0.4-0.9,11.8-0.9,11.8-0.9z`}/>
+export interface PlaygroundModeIconProps {
+  disabled: boolean;
+  style: object;
```

Comment:
nitpick: I think you can add a React.CSSProperties typedef to this, like we do here https://github.com/all-of-us/workbench/blob/322e08140e18415d2cc7115fe919c922551f0e4b/ui/src/app/components/spinners.tsx#L41